packages/shared/lib/clients/provider.client.ts

```diff
@@ -156,6 +159,29 @@ class ProviderClient {
         throw new NangoError('facebook_token_request_error');
     }
 
+    private async createTiktokToken(tokenUrl: string, code: string, clientId: string, clientSecret: string): Promise<object> {
+        try {
+            const body = {
+                secret: clientSecret,
+                app_id: clientId,
+                auth_code: code
+            };
+
+            const response = await axios.post(tokenUrl, body);
+
+            if (response.status === 200 && response.data !== null) {
+                return {
+                    access_token: response.data.data['access_token'],
+                    advertiser_ids: response.data.data['advertiser_ids'],
+                    scope: response.data.data['scope']
+                };
+            }
+            throw new NangoError('tiktok_token_request_error');
```

Comment:
it seems we never throw with the error, isn't that making things complicated for debugging?

Reply:
If it isn't a 200 for some reason this would be thrown. It would be good to know if it isn't a 200 and we will get notified by the user that the connection attempt failed + see something in logs.